api/handlers/middleware.go

```diff
@@ -47,3 +47,30 @@ func LoggingMiddleware(log *logrus.Logger) gin.HandlerFunc {
 		}
 	}
 }
+
+// HasOneRoleOf returns a gin handler that checks the request against the given role
+func HasOneRoleOf(roles ...security.Role) gin.HandlerFunc {
+	return func(c *gin.Context) {
+		rolesRaw, ok := c.Get(KeyRoles)
+		rolePolicy, castok := rolesRaw.(security.RolePolicy)
+		if !ok && !castok && rolePolicy != nil {
+			// Bypass roles check if the configuration has not been properly set
```

Comment:
Maybe not a good idea to "bypass security checks" if configuration is not good

Reply:
👍 default behavior is inverted